.github/workflows/test-python.yaml

```diff
@@ -0,0 +1,41 @@
+name: Python Test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Install Packages
+        run: |
+          pip install grpcio-testing==1.40.0
+          pip install pytest==5.1.2
```

Comment:
How about make a `requirements.txt` including this packages at the source root directory ?